cmd/podman/commit.go

```diff
@@ -83,7 +84,7 @@ func commitCmd(c *cli.Context) error {
 			return errors.Errorf("messages are only compatible with the docker image format (-f docker)")
 		}
 	case "docker":
-		mimeType = buildah.Dockerv2ImageManifest
+		mimeType = manifest.DockerV2Schema1MediaType
```

Comment:
This is changing the value from `...manifest.v2...` to `...manifest.v1...`:

```console
$ git grep V2S2MediaTypeManifest vendor | grep 'types\.go\|Dockerv2ImageManifest'
vendor/github.com/projectatomic/buildah/docker/types.go:const V2S2MediaTypeManifest = "application/vnd.docker.distribution.manifest.v2+json"
vendor/github.com/projectatomic/buildah/image.go:       Dockerv2ImageManifest = docker.V2S2MediaTypeManifest
$ git grep 'DockerV2Schema1MediaType =' vendor
vendor/github.com/containers/image/manifest/manifest.go:        DockerV2Schema1MediaType = "application/vnd.docker.distribution.manifest.v1+json"
```

I think you want to use `DockerV2Schema2MediaType`:

```console
$ git grep 'application/vnd.docker.distribution.manifest.v2' vendor/github.com/containers/image/manifest            
vendor/github.com/containers/image/manifest/manifest.go:        DockerV2Schema2MediaType = "application/vnd.docker.distribution.manifest.v2+json"
```

(and similarly in the other file).

Reply:
good catch ... fixed ... need caffeine